### PR TITLE
chore: use grpcio-tools protoc function instead of modules

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**
#149

### CODE REVIEW

**Review by arithmetic1728 — APPROVED** [review]

File: src/detector/loader.py [hunk 1]
```diff
@@ -46,7 +47,7 @@ def __init__(
         self.descriptor_set = descriptor_set
         self.proto_files = proto_files
         self.include_source_code = include_source_code
-        self.protoc_binary = protoc_binary or self.PROTOC_BINARY
+        self.protoc_binary = protoc_binary or "grpc_tools.protoc"
```
Comment (alexander-fenster) [inline src/detector/loader.py:50]: I would better just leave it as `None`:

```
self.protoc_binary = protoc_binary
```

and then

```
if self.protoc_binary is None:
  # do stuff
```

This is better than introducing a magic string constant `"grpc_tools.protoc"`.

Comment (alexander-fenster) [reply src/detector/loader.py:50]: Note that you don't actually need to pass `grpc_tools.protoc` to the list of parameters, it perfectly works without it. So in line 63, you only need to append `self.protoc_binary` to the list of parameters if it's set to a real binary path.

**Review by alexander-fenster — APPROVED** [review]
LGTM with one minor comment. Thank you for doing this!

File: src/detector/loader.py (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply src/detector/loader.py:50]: Thanks for the advice! the "grpc_tools.protoc" removal emits errors on my side, about not finding a specific file in "--proto_path". 
How to reproduce > run "python -m unittest discover test/comparator" one of the failed command is like below:
```
src.detector.loader._ProtocInvokerException: Protoc command to load the descriptor set fails. ['--proto_path=/usr/local/google/home/xiaozhenliu/Downloads/breaking-change-detector-develop/proto-breaking-change-detector/test/testdata/protos/example/', '--proto_path=/usr/local/google/home/xiaozhenliu/Downloads/breaking-change-detector-develop/proto-breaking-change-detector/api-common-protos', '--proto_path=/usr/local/google/home/xiaozhenliu/Downloads/breaking-change-detector-develop/proto-breaking-change-detector/protobuf/src', '--include_source_info', '--include_imports', '/usr/local/google/home/xiaozhenliu/Downloads/breaking-change-detector-develop/proto-breaking-change-detector/test/testdata/protos/example/wrappers.proto', '--descriptor_set_out=/tmp/tmp8zpgth52']
```
I copied the parameters and run in terminal and it fails for "/usr/local/google/home/xiaozhenliu/Downloads/breaking-change-detector-develop/proto-breaking-change-detector/test/testdata/protos/example/wrappers.proto: File does not reside within any path specified using --proto_path (or -I).  You must specify a --proto_path which encompasses this file.  Note that the proto_path must be an exact prefix of the .proto file names -- protoc is too dumb to figure out when two paths (e.g. absolute and relative) are equivalent (it's harder than you think)."

```
from grpc_tools import protoc

command=["the/above/paratemers"]
protoc.main(command) // fails.
command=["grpc_tools.protoc", "append/the/above/parameters"] // works
```

Comment (alexander-fenster) [reply src/detector/loader.py:50]: I see. Let's just make `"grpc_tools.protoc"` a constant somewhere in that file so we did not have a duplicated string in the code.

Comment (xiaozhenliu-gg5) [reply src/detector/loader.py]: done.

**Pull request merged**